client/state/reader/streams/reducer.js

```diff
@@ -133,7 +132,7 @@ export const pendingItems = ( state = PENDING_ITEMS_DEFAULT, action ) => {
 			}
 
 			maxDate = moment( streamItems[ 0 ].date );
-			minDate = moment( last( streamItems ).date );
+			minDate = moment( streamItems?.at( -1 )?.date );
```

Comment:
Just on the previous line we're using `streamItems` as if it's an array, so perhaps we don't really need the first optional chaining?